survey.py

```diff
@@ -12,16 +12,21 @@ def get_args():
         print("Unknown Error! See full details below")
         raise
 
-
 def main():
     filename = get_args() #Get filename for sys arg
-    file_exists(filename) # Check if file exists
-    final_list = get_input(filename)
-    capitalized_list = capitalize_names(final_list)
-    validated_list = validate(capitalized_list)
-    writedata(validated_list)
+    
+    if filename != 'clean_results.csv':
+        file_exists(filename) # Check if file exists
+        final_list = get_input(filename)
```

Comment:
Code itself looks good to me, just would maybe choose a different variable name here for clarity - something like initial_list? Since your "final list" is really your validated_list?